src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {
+  Breadcrumbs as BreadcrumbsMUI,
+  Link,
+  useTheme,
+  Button,
+  Grid,
+  Box,
+  Typography
+} from '@mui/material';
+import { ArrowBack } from '@mui/icons-material';
+import { useTranslation } from 'react-i18next';
+import { useMatches } from 'react-router-dom';
+
+export interface BreadcrumbPath {
+  backButton?: boolean;
+  elements: BreadcrumbElement[];
+  routeName: string;
+}
+
+export interface BreadcrumbElement {
+  name: string;
+  fontWeight: number;
+  color?: string;
+  href?: string;
+}
+
+export interface CrumbObject {
+  crumb: BreadcrumbPath;
+}
+
+const Breadcrumbs = ({ separator }: { separator: React.ReactElement }) => {
+  const theme = useTheme();
+  const { t } = useTranslation();
+  const matches = useMatches();
+  const crumbs = (matches.find((match) => Boolean(match.handle))?.handle as CrumbObject)?.crumb;
+
+  const show = crumbs && crumbs.elements.length > 1;
+
+  return (
+    show && (
+      <Grid container maxHeight={40}>
+        {crumbs.backButton && (
+          <Grid item>
+            <Box>
+              <Button
+                role="button"
+                aria-label="back-button"
```

Comment:
All the label must be translated using the correct function and definition in translation file

Reply:
Should I create an entry for aria labels?

Reply:
In the translation file put a string like `Torna indietro` and then in the aria-label `{t("translationfile.BackButton")}, see the [spec](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)